src/Microsoft.IdentityModel.Tokens/Encryption/AuthenticatedEncryptionProvider.cs

```diff
@@ -32,6 +33,20 @@ private struct AuthenticatedKeys
         private DecryptionDelegate DecryptFunction;
         private EncryptionDelegate EncryptFunction;
         private const string _className = "Microsoft.IdentityModel.Tokens.AuthenticatedEncryptionProvider";
+        internal const string _skipValidationOfAuthenticationTagLength = "Switch.Microsoft.IdentityModel.SkipAuthenticationTagLengthValidation"; 
+
+        /// <summary>
+        /// Mapping from algorithm to the expected authentication tag length.
+        /// </summary>
+        internal static readonly Dictionary<string, int> ExpectedAuthenticationTagBase64UrlSizeInBytes = new()
+        {
+            { SecurityAlgorithms.Aes128Gcm, 16 },
```

Comment:
@brentschmaltz while testing, I discovered that the auth length issue does not repro with the AesGcm algos - do we anticipating needing these values in the future, or shall I remove them?

Reply:
I went ahead and removed the mappings as there are similar ones in SymmetricSignatureProvider. See this [comment](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2569/files#r1583822363)

Reply:
@kellyyangsong I can't see the comment anymore. Does this PR apply only to AesCbc?